### DISCUSSION TITLE
insight/user-guide/alert-center/alert-policy

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3171] [Announcements]
# insight/user-guide/alert-center/alert-policy

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/insight/user-guide/alert-center/alert-policy.html

**Comment by samzong** [comment]
建议 告警策略的 YAML 需要有示例.
